contrib/epee/include/net/levin_protocol_handler_async.h

```diff
@@ -585,10 +585,11 @@ class async_protocol_handler
           m_cache_in_buffer.erase(sizeof(bucket_head2));
           m_state = stream_state_body;
           m_oponent_protocol_ver = m_current_head.m_protocol_version;
-          if(m_current_head.m_cb > max_packet_size)
+          const size_t max_bytes = m_connection_context.get_max_bytes(phead.m_command);
+          if(m_current_head.m_cb > std::min<size_t>(max_packet_size, max_bytes))
```

Comment:
A check needs to be added around 472 - the fragmented packet case where tor/i2p transactions are sent in fixed sized messages at randomized intervals.

Reply:
I'm not sure what check you want, I di not understand that cusrom code.

Reply:
The `m_current_header` has the levin header just like it does on this line. So the check is identical.